common/lib/kernel/vfio_device.cpp

```diff
@@ -247,6 +247,51 @@ bool Device::pciEnable() {
   return true;
 }
 
+int Device::platformInterruptInit(int efds[]){
+  size_t efd_cnt = 0; //FIXME: efds currently has a fixed size of 32. But we don't check that and there may be more efds.
```

Comment:
Please add a space between `//` and the comment itself:

```suggestion
  size_t efd_cnt = 0; // FIXME: efds currently has a fixed size of 32. But we don't check that and there may be more efds.
```